services/filter/src/filter/main.py

```diff
@@ -0,0 +1,7 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright 2022 The HuggingFace Authors.
```

Comment:
nit: replace 2022 with 2023 in the new files

Reply:
Done.